README.md

```diff
@@ -1 +1,42 @@
 # bit-docs-prettify
+
+A [bit-docs](https://github.com/bit-docs/bit-docs) plugin that makes source-code snippets in HTML prettier.
+
+Powered by https://github.com/google/code-prettify.
+
+## Quick Start
+
+Add `bit-docs-prettify` as a dependency to your project's `package.json` under the `bit-docs` section:
+
+```json
+{
+  "name": "your-project",
+  ...
+  "bit-docs": {
+    "dependencies": {
+      "bit-docs-prettify": "*"
+    },
+    ...
+  }
+}
+```
+
+Now run `./node_modules/bit-docs/bin/bit-docs` in your project to finish adding `bit-docs-prettify`.
+
+Your `bit-docs` enabled project should now have syntax highlighting of source-code snippets!
```

Comment:
Do you want a bullet list of every default language supported, like:

  - bsh
  - c
  - cc
  - cpp
  - cs
  - csh
  - cyc
  - cv
  - htm
  - html
  - java
  - js
  - m
  - mxml
  - perl
  - pl
  - pm
  - py
  - rb
  - sh
  - xhtml
  - xml
  - xsl

Or just a comma-separated list, like:


Supported Languages:

    bsh, c, cc, cpp, cs, csh, cyc, cv, htm, html, java, js, m, mxml, perl, pl, pm, py, rb, sh, xhtml, xml, xsl